admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx

```diff
@@ -58,16 +66,21 @@ export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (p
                             id='display-name'
                             name="displayName"
                         /></FormGroup>
-                    <FormGroup label='Description' fieldId='description'
-                        helperText='Optional short description that appears in the UI
-                                                to help admin decide how to notify users.'>
-                        <TextArea
-                            type='text'
-                            value={ eventType.description }
-                            onChange={ handleChange }
-                            id='description'
-                            name="description"
-                        /></FormGroup>
+                    <FormGroup label='System Behavior Group' fieldId='system behavior group'
+                        helperText='Behavior groups are made up of action/recipient pairings that allow you to configure which
+                        notification actions different users will be able to receive. Once you have created
+                        a behavior group, you can assign it to an event type.'>
+                        <FormSelect
```

Comment:
We will need to use other component - it's possible to assign multiple behavior groups to a single event type - although maybe we don't want to overuse this. Lets leave it for now with 1 default behavior group per event.